documentation/API_documentation/Network Slice Booking API Design.md

```diff
@@ -0,0 +1,77 @@
+# Network Slice Booking API Design
+
+## **Use Cases**
+Scenario can be  B2B, B2C, B2B2C
+
+1. Scenario 1 - B2B
+
+   Sell Slice Directly
+
+Customer: toB Enterprise Customer
+
+Seller: Operator 
+
+Mode: Operator provide slice services by NaaS API to enterprise customers directly. 
+
+Example Use Cases: In the past/current, most of  current slice customer need to order slice from operators directly through operator customer managemer. This takes time from 2 weeks to 2 month. Those kind of customers may renew, reopen, modify, close the slice service using NaaS Slice APIs. 
+
+
+
+
+2. Scenario 2 - B2C
+
+OTT act as a Platform to Sell Slice
```

Comment:
'OTT' typically not refers to an actor. What is meant here? Does OTT resolve to Over-The-Top?

Is this here an Application Provider or an Application Service Provider, who is the customer (not OTT)?

Reply:
How was this comment resolved?